src/test/java/com/google/sps/CommentRetrievalTest.java

```diff
@@ -0,0 +1,87 @@
+// Copyright 2020 Google LLC
```

Comment:
Big concerns here:
looks like the test did not pass on my local: there is MissingMethodInvocationException:(

Reply:
Is it working on yours? if so, it might be my local setup issues...

Reply:
I actually do get a Compilation failure but it's from CommentAnalysisTest and therefore outside the scope of this pr.